item.rb

```diff
@@ -13,6 +13,11 @@ def add_label(label)
     @label = label
   end
 
+  def add_genre(genre)
+    @genre = genre
+    genre.items << self
```

Comment:
You don't need to do this if you are setting it in the `Genre` class when you `add_item`.
Here `add_genre` serves as a setter only

Reply:
Hello @michael-duke ,

This has been updated.

Reply:
Well done